ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh

```diff
@@ -68,7 +68,7 @@ case "$FN" in
 	1)
 		echo "$MINUTE $HOUR * * *"
 		;;
-	2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)
+	2|3|4|5|6|7|10|14|28|30)
```

Comment:
I suggest to keep 21 here. We may use it as one Sprint is 3 weeks (21 days). If any job need to follow Sprint, we have this option here.

Reply:
It's already explained in the description.

Reply:
I think this code is just providing option of f21. It's not saying all job using f21 right now. Am I right? I just wan to keep this option exist. If in future, we need f21, we can directly use.

Reply:
f21 does not do what we expected it to do. That's why we need to remove it. We do not wan to let people choose f21 as they wish it to execute every 3 weeks, but actually it executes every month.
We may consider to add it back once we found a solution to let jobs execute at every 3 weeks.

Reply:
My understanding is cron limitation. I am ok with you approach right now.